portfolio/src/main/webapp/style.css

```diff
@@ -135,20 +169,57 @@ body,html {
   
 }
 
+a { color: white; text-decoration: none; }
+
+.arrow {
+  text-align: center;
+  margin: 0;
+  border: none;
+}
+.bounce {
+  -moz-animation: bounce 2s infinite;
+  -webkit-animation: bounce 2s infinite;
+  animation: bounce 2s infinite;
+}
+
+@keyframes bounce {
+  0%, 20%, 50%, 80%, 100% {
+    transform: translateY(0);
+  }
+  40% {
+    transform: translateY(-30px);
+  }
+  60% {
+    transform: translateY(-15px);
+  }
+}
+
+.footer {
+    text-align: center;
+    padding: 20px;
+    background-color:#f8f3f1
+
+}
 h1 {
     text-align: center;
-    font-size: 50pt;
+    font-size: 30pt;
```

Comment:
It's okay to use whatever measurement you want for the size/height/width but ideally, we wanna align them to the same measurement, in my daily work, we use px most and sometimes %, just FYI.